src/bo4e/bo/marktlokation.py

```diff
@@ -98,6 +103,7 @@ def _get_checksum(malo_id: str) -> str:
         # start counting at 1 to be consistent with the above description
         # of "even" and "odd" but stop at tenth digit.
         for i in range(1, 11):
+            # pylint: disable=invalid-name
```

Comment:
Welcher Name ist hier invalid?

Reply:
vermute `s`. könnte man in `digit` umbenennen.

Reply:
25a7f9727da8b27b3af32552da68054ff874d537
done